File: 240624/두 숫자의 차의 최솟값/minimum-difference-between-two-numbers.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {


        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();

        int arr[] = new int[a];

        for(int i = 0; i < a; i++){
            arr[i] = sc.nextInt();
        }

        int min = 99999;

        for(int i = 0; i < a; i++){
            for(int j = i+1; j < a; j++){
                min = Math.min(Math.abs(arr[i]-arr[j]), min);
            }
        }

        System.out.println(min);

    }
}

File: 250305/두 수의 차의 최솟값/minimum-difference-between-two-numbers.py
n = int(input());

arr = list(map(int, input().split()));

min_val = 9999999;

up = 0;
down = 0;


for a in arr :
    for b in arr :
        if a == b :
            continue;
        min_val = min(min_val, abs(a-b));

print(min_val);

        
